src/gnn_reco/data/utils.py

```diff
@@ -9,6 +9,46 @@
 from typing import List, Tuple
 import sqlite3
 
+
+def GetDesiredEventNumbers(db_path, desired_size, fraction_noise = 0, fraction_nu_e = 0, fraction_muon = 0, fraction_nu_mu = 0, fraction_nu_tau = 0):
```

Comment:
```suggestion
def get_desired_event_numbers(db_path, desired_size, fraction_noise=0, fraction_nu_e=0, fraction_muon=0, fraction_nu_mu=0, fraction_nu_tau=0):
```

Reply:
I would probably reorder the arguments so as to keep the three neutrino flavours next to each other rather than having the muon in the middle, e.g.:`noise`,  `muon`, `nu_e`, `nu_mu`, `nu_tau`.

Reply:
The name of this function could probably also be a bit more descriptive, as "desired" is quite subjective/contextual. I think it is perfectly fine for now, and the intention is very clear from the arguments, just something to think about going forward.